app/views/nouns/_form.html.haml

```diff
@@ -2,6 +2,7 @@
   = two_column_card t('words.show.noun.title'), t('words.show.noun.description') do
     = box do
       = f.input :name
+      = f.input :with_tts, as: :boolean
```

Comment:
That's very prominent here:

![image](https://user-images.githubusercontent.com/1394828/208094150-7a63a6c4-e3ee-4f72-8b0a-c512887572ca.png)

What about moving it to the syllables instead, in `app/views/words/_general.html.haml`? That way, we only would have it in one location for all word types.

![image](https://user-images.githubusercontent.com/1394828/208094351-574749b9-c14a-4ee4-80c1-bc1bd1955300.png)

Same for the form, this could be moved into `app/views/words/_general_form.html.haml`.

You could remove the `as: :boolean`, as simple_form will notice it, unless you added it for clarity.